src/main/java/ingest/messaging/IngestThreadManager.java

```diff
@@ -57,169 +47,67 @@
  */
 @Component
 public class IngestThreadManager {
-	private static final String INGEST_TOPIC_NAME = IngestJob.class.getSimpleName();
 	@Autowired
 	private PiazzaLogger logger;
 	@Autowired
 	private IngestWorker ingestWorker;
 
-	@Value("${vcap.services.pz-kafka.credentials.host}")
-	private String KAFKA_HOSTS;
-	@Value("#{'${kafka.group}' + '-' + '${SPACE}'}")
-	private String KAFKA_GROUP;
 	@Value("${SPACE}")
 	private String SPACE;
 
-	private Producer<String, String> producer;
-	private Map<String, Future<?>> runningJobs;
-	private final AtomicBoolean closed = new AtomicBoolean(false);
-
+	private Map<String, Future<?>> runningJobs = new HashMap<String, Future<?>>();
```

Comment:
We can eliminate the redundancy with: 
`	private Map<String, Future<?>> runningJobs = new HashMap<>();`